src/stirling/source_connectors/perf_profiler/perf_profiler_bpf_test.cc

```diff
@@ -454,14 +454,14 @@ TEST_F(PerfProfileBPFTest, PerfProfilerCppTest) {
 
 // TODO(jps/oazizi): This test is flaky.
 TEST_F(PerfProfileBPFTest, DISABLED_GraalVM_AOT_Test) {
```

Comment:
We need to remove the `DISABLED_` prefix and this TODO, right?

Reply:
Testing was done using a different branch that re-based the next diff (which removes DISABLED_) onto this diff.